features/56-login-with-custom-token/api.md

```diff
@@ -0,0 +1,160 @@
+## Feature Overview
+
+Allow user to integrate their authentication server to skygear server by logging in with custom token
+
+## Guides
+
+### Sample Codes (Client SDK)
+
+**JS**
+
+```js
+skygear.auth.loginWithCustomToken("<custom_token>")
+.then(function(skygearUser) {
+
+}).catch(function(error) {
+
+});
+```
+
+**iOS (swift)**
+
+```swift
+SKYContainer.default().auth.login(withCustomToken: "<custom_token>") { (user, error) in
+    if error != nil {
+        print("error login with custom token: \(error)")
+        return
+    }
+    print("sign up successful")
+    print("user record: \(user)")
+    print("email: \(user!["email"])")
+}
+```
+
+**Android (java)**
+
+```java
+skygear.getAuth().loginWithCustomToken("<custom_token>", new AuthResponseHandler() {
+    @Override
+    public void onAuthSuccess(Record user) {
+        Log.i("Skygear loginWithCustomToken", "onAuthSuccess: Got token: " + user.getAccessToken());
+    }
+
+    @Override
+    public void onAuthFail(Error error) {
+        Log.e("Skygear loginWithCustomToken", "onAuthFail: Reason: " + error.getMessage());
+    }
+});
+```
+
+### Sample Codes (User's server)
+
+1. Get the token secret from portal, and setup environment variable in your server `SKYGEAR_CUSTOM_TOKEN_SECRET`
+
+2. Generate create token
+
```

Comment:
I would add a description on where to add this line. Normally during the user login. probably suggest the developer return the custom token to client-side SDK during signup.